app/index.html

```diff
@@ -177,129 +177,158 @@ <h1 ng-bind-template="{{'EMAIL_SIGNUP' | translate}}">Sign Up For Our Newsletter
                         </div>
                     </form>
                 </div>
+        </section>
+        <section id="press">
+          <ul class="press-images">
+            <li><a class="press-image" target="_blank" href="http://techpresident.com/news/wegov/24444/state-department-funded-lantern-next-bigger-better-tor">
+              <img src="/images/press/tech-president.png" alt="Could State Department Funded Lantern Be Bigger, Better Tor?">
+        </a></li>
+        <li><a class="press-image"  target="_blank" href="http://www.scmp.com/news/china-insider/article/1372661/use-lantern-software-means-view-banned-websites-grows-china">
+          <img src="/images/press/scmp.png" alt="US-funded Lantern program allows Chinese to dodge Great Firewall and view banned websites">
+        </a></li>
+        <li><a class="press-image"  target="_blank" href="http://www.washingtonpost.com/blogs/the-switch/wp/2014/08/21/using-peer-to-peer-technology-to-crowdsource-a-way-around-online-censorship/">
+          <img src="/images/press/the-washington-post.png" alt="Using peer-to-peer technology to crowdsource a way around online censorship">
+        </a></li>
+        <li><a class="press-image"  target="_blank" href="https://gigaom.com/2014/08/14/anti-censorship-app-lantern-wants-to-become-the-setihome-of-free-speech/">
+          <img src="/images/press/gigaom.png" alt="Anti-censorship app Lantern wants to become the SETI@home of free speech">
+        </a></li>
+        <li><a class="press-image"  target="_blank" href="http://www.dailydot.com/politics/lantern-online-censorship/">
+          <img src="/images/press/the-dialy-dot.png" alt="How you can help demolish the Great Firewall of China from the comfort of your living room">
+        </a></li>
+        <li><a class="press-image"  target="_blank" href="http://www.lemonde.fr/pixels/article/2014/08/21/lantern-le-logiciel-qui-veut-eradiquer-la-censure-d-internet_4473868_4408996.html">
+          <img src="/images/press/le-monde.png" alt="Lantern, le logiciel qui veut éradiquer la censure d'Internet">
+        </a></li>
+      </ul>
+      <div class="praise">
+        <div class="user-praise-from-scmp" ng-bind-template="{{'USER_PRAISE_FROM_SCMP' | translate}}">&ldquo;Lantern is the most intuitive tool to access the internet freely&ddquo;</div>
+        <div class="user-scmp">
+        <span ng-bind-template="{{'USER_QUOTED' | translate}}"> - Lantern user quoted in the</span>
+        <a href="http://www.scmp.com/news/china-insider/article/1372661/use-lantern-software-means-view-banned-websites-grows-china" ng-bind-template="{{'SCMP' | translate}}">South China Morning Post</a>
```

Comment:
I would add a target="_blank" attribute here so that the article opens in a new tab?